### PR TITLE
Use plan display name + show original name at netuney hatochnit

### PR DESCRIPTION
**Pull request by GalGend**
I added the original image 

resolving #371

### CODE REVIEW

File: client/src/pages/Plan/common/Panels/data.jsx [hunk 1]
```diff
@@ -76,6 +77,12 @@ export const DataPanel = ({
 						</Moment>
 					</li>
 				</SC.Datalist>
+				{originalName && (
```
Comment (florpor) [inline client/src/pages/Plan/common/Panels/data.jsx:80]: is this outside of the Datalist component on purpose? (unlike all above list items)

Comment (GalGend) [reply client/src/pages/Plan/common/Panels/data.jsx:79]: fixed

File: client/src/pages/Plan/common/Panels/data.jsx [hunk 2]
```diff
@@ -76,6 +77,12 @@ export const DataPanel = ({
...
+						<li key="jurisdiction">
+                           שם תוכנית מקורי:
```
[the remark this answers is not in the document]
Comment (GalGend) [reply client/src/pages/Plan/common/Panels/data.jsx]: done

File: client/src/pages/Plan/common/Panels/data.jsx [hunk 3]
```diff
@@ -76,6 +77,12 @@ export const DataPanel = ({
 						</Moment>
 					</li>
 				</SC.Datalist>
+				{originalName && (
+						<li key="jurisdiction">
```
[the remark this answers is not in the document]
Comment (GalGend) [reply client/src/pages/Plan/common/Panels/data.jsx]: done

**Review by florpor — APPROVED** [review]

**Pull request merged**